chrome/content/zotero/xpcom/intl.js

```diff
@@ -82,6 +82,13 @@ Zotero.Intl = new function () {
 			let regexp = /<!ENTITY ([^\s]+)\s+"([^"]+)/g;
 			let regexpResult;
 			while (regexpResult = regexp.exec(localeXML)) {
+				let bundleString;
+				try {
+					bundleString = bundle.GetStringFromName(regexpResult[1]);
+				} catch (e) {}
+				if (bundleString) {
+					throw new Error(`Found a duplicate string ${regexpResult[1]} for both bundle and dtd strings`)
+				}
```

Comment:
We should just add this to a test. It's only ~4 milliseconds for me, but no reason to do it on every startup, and milliseconds add up eventually.

Reply:
We're initializing the strings here anyway so I didn't think it egregious. If it's a test you'll end up not noticing you've added a duplicate until pushing to github and the test suite running, which is like a whole lot more compute, if we're concerned about that.

Reply:
It's totally fine, but there's a (very small) measurable difference for just this block, so we might as well avoid it.

Given that there was a single duplicate in 15 years of strings, I think it's fine for it not to be caught until CI (and that's only necessary if you mess it up in the direction where you wouldn't immediately notice it in the UI you're working on).